### DISCUSSION TITLE
Questions about overlapping arc

### DISCUSSION THREAD

**Discussion by LNTU-YS** [groops-devs/groops#293] [Q&A] [answered]
Hello,

Can we achieve LEO POD across two days in GROOPS? I want to conduct overlapping arc analysis, but I found that when I extended the time setting to the second day, the result of POD was still only for the first day. What should I do if I want to implement a plan that takes 27 hours from DOY1 0:00 to DOY2 3:00 or 24 hours from DOY1 12:00 to DOY2 12:00?

Thank you in advance for your feedback and help.
With Kind Regards
YUAN

**Comment by LNTU-YS** [comment]
Or, in a 24-hour day, I only select 15 hours for the experiment, such as DOY 1 0:00-15:00. Is it possible to achieve this?

**Comment by tmayerguerr** [comment, marked as answer]
Hi YUAN,

One way to do this:

- Create a daily loop and then define additional variables such as `loopTimeArc=loopTime-3/24` and `loopTimeEndArc=loopTimeEnd+3/24` for additional three hours before and after. 
- With [InstrumentConcatenate](https://groops-devs.github.io/groops/html/InstrumentConcatenate.html) you can merge the data from the current day, the day before and the day after
- With [InstrumentSynschronize](https://groops-devs.github.io/groops/html/InstrumentSynchronize.html)->timeIntervals you can cut out your interval ([uniformInterval](https://groops-devs.github.io/groops/html/timeSeriesType.html) with `loopTimeArc`, `loopTimeEndArc`, and `intervalCount=1`).

regards
Torsten

**Reply by LNTU-YS** [reply]
Hi Torsten, 

Thank you very much for your help. I have followed your instructions and tried POD successfully. I hope I’m not bothering you, but I’d love to ask you another question if you have the time. 

How do I concatenate OSB information when I use OSB files for fixed ambiguity? If I use InstrumentConcatenate , it will encounter errors "file type is 'gnssSignalBias' but must be 'instrument' or 'matrix' continue...". So I want to know if you have any good ideas.
![image](https://github.com/user-attachments/assets/ae4d6d90-01f5-4be1-8acb-d3e31d5c3de1)


Thank you in advance for your feedback and help.
With Kind Regards
YUAN

**Comment by tmayerguerr** [comment]
Ah, sorry, I had forgotten the bias. With GROOPS, the bias must be constant over the processing period. I think it should work if you just take the file from the middle day.
Best regards
Torsten

**Reply by LNTU-YS** [reply]
Hi Torsten,

The time I am currently processing data settings is DOY 1 0:00-DOY 2 1:00, totaling 25 hours. In fact, I had already attempted this before. I only use first day's OSB. And I processed data for 30 days. But when I analyzed the overlapping arc, I obtained a fixed ambiguity orbit that was worse than the float solution. Do you have any other good ideas?

With Kind Regards
YUAN

**Reply by tmayerguerr** [reply]
Hi YUAN,
I should probably think before I reply. At least the phase bias must be compatible with the estimated transmitter clocks. I'm running out of ideas at the moment. So this doesn't work with GROOPS. In the long term, it would be good to improve GROOPS in this direction. But I don't see an easy solution and we probably don't have time to tackle this problem in the next six months.
best regards
Torsten

**Reply by LNTU-YS** [reply]
Hi Torsten,

Thank you very much for your reply. I will think of other ways to achieve it.

With Kind Regards
YUAN
